module/Database/src/Form/Member.php

```diff
@@ -328,10 +332,33 @@ public function getInputFilterSpecification(): array
                 'iban' => [
                     'required' => true,
                     'validators' => [
-                        ['name' => Iban::class],
+                        [
+                            'name' => Iban::class,
+                            'options' => [
+                                'allow_non_sepa' => false,
+                                'messages' => [
+                                    Iban::NOTSUPPORTED     =>
+                                        $this->translator->translate('IBAN starts with unknown country code.'),
+                                    Iban::SEPANOTSUPPORTED =>
+                                        $this->translator->translate('IBANs from countries that do not participate in SEPA are not supported.'),
+                                    Iban::FALSEFORMAT      =>
+                                        $this->translator->translate('The input has a false IBAN format.'),
+                                    Iban::CHECKFAILED      =>
+                                        $this->translator->translate('The input has failed the IBAN check.'),
+                                ],
+                            ],
+                        ],
                     ],
                     'filters' => [
                         ['name' => StringTrim::class],
+                        ['name' => StringToUpper::class],
+                        [
+                            'name' => PregReplace::class,
+                            'options' => [
+                                'pattern' => '/\\s/',
+                                'replacement' => '',
+                            ],
+                        ],
```

Comment:
Perhaps we should use the `Alnum` filter?

Reply:
[8fb8edb](https://github.com/GEWIS/gewisdb/pull/200/commits/8fb8edb178737bd2be7cd72116765a9b0f1725f2)